driver/outer_driver.hpp

```diff
@@ -0,0 +1,517 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2023 Advanced Micro Devices, Inc.
```

Comment:
```suggestion
 * Copyright (c) 2024 Advanced Micro Devices, Inc.
```

Reply:
I can easily find Copyright (c) 2023 in 4 files in this PR. Please fix them all.

Reply:
I followed your suggestion.